packages/terra-form-validation/docs/ValidationEvents.md

```diff
@@ -0,0 +1,103 @@
+# Validation Events
+
+The default behavior of *react-final-form* is to validate dynamically (while the user is interacting with the form, and on form load). To use this behavior, simply use the error key of the meta data passed to an individual form component, and pass it to the appropriate InputField for TextareaField, like so below.
```

Comment:
nit: The default behavior of *react-final-form* is to **dynamically** validate ~dynamically~ **,meaning it validates the content** ~(~ while the user is interacting with the form ~,~ and **when a** ~on~ form load**s**

Reply:
and pass it to the ~appropriate~ **error prop of either the ** InputField ~for~ **or** TextareaField, like ~so~ **in the example** below.